photon-core/src/main/java/org/photonvision/vision/frame/consumer/FileSaveFrameConsumer.java

```diff
@@ -117,4 +126,15 @@ public void updateCameraNickname(String newCameraNickname) {
         this.subTable = rootTable.getSubTable(this.camNickname);
         resetCommand();
     }
+
+    private String getMatchData() {
+        /**
+         * Returns the match Data collected from the NT.
+         * eg : Q58 for qualfication match 58.
+         * If not in event, returns N/A-0
+         */
+        String matchType = matchTypes[(int)ntMatchType.getDouble(0)];
```

Comment:
This has the possibility to throw an index out of bounds right?

Reply:
As long as the user won't change it manually, no.
data published to this entry can be only in bounds.

Reply:
it's still possible a user does that , knowingly or unknowingly, and crashes the program. We can't let users crash.

Reply:
well, it does make sense - I will add validation.